dcc-portal-server/src/main/java/org/icgc/dcc/portal/server/config/ServerConfig.java

```diff
@@ -161,4 +162,8 @@ public AuthProperties authProperties() {
     return properties().getAuth();
   }
 
+  @Bean
+  public SoftwareProperties softProperties() {
+    return properties().getSoft();
```

Comment:
Lets call this `getSoftware`. If we ever use the yaml file, I think `soft:` will be unintuitive. FYI - I am not a fan of `elastic` either.